lib/workers/repository/dependency-dashboard.ts

```diff
@@ -19,7 +21,8 @@ function parseDashboardIssue(issueBody: string): DependencyDashboard {
     const re = new RegExp(checkMatch);
     checked.forEach((check) => {
       const [, type, branchName] = re.exec(check);
-      dependencyDashboardChecks[branchName] = type;
+      const cleanedUpBranchName = branchName.replace(RE_MULTIPLE_DASH, '-'); // Backward compatibility with versions which didn't remove chained dashes
+      dependencyDashboardChecks[cleanedUpBranchName] = type;
```

Comment:
@rarkins Do we really need this? Should be a very short period when this can happen. The only think i can imagin is that some checked branches are getting lost. 🤔 

Renovate will close and reopen those branches / prs anyways

Reply:
We don't need this.